liiatools_pipeline/sensors/concatenate_sensor.py

```diff
@@ -0,0 +1,22 @@
+from dagster import RunRequest, RunsFilter, DagsterRunStatus, sensor
+from liiatools_pipeline.jobs.common_la import move_current, concatenate
+
+
+@sensor(job=concatenate)
+def concatenate_sensor(context):
```

Comment:
@DJavur Could you also rename the function ssda903_concatenate_sensor so that it's consistent with the rest of the pipelines?  Thank you!

Reply:
- Discussed with @cyramic last week that the names can remain the same since we will be doing away with separate cin & 903 files to have a common file for both. 
- Made the changes last week to include default status to be running for the sensors.